presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -25,17 +23,8 @@
 
 public final class EnvironmentOptions
 {
-    @Option(name = "--hadoop-base-image", title = "image", description = "Hadoop base image")
```

Comment:
This limits the expressiveness of `env up`. Are you sure people did not depend on possibility to build test environment bit by bit?

Reply:
That's why I've created `config-env-based` that allows passing those values using ENV. There is also a possibility to create `EnvironmentConfig` with desired values and use it with `env up`.

Reply:
So actually it makes extending configuration properties for environments easier than before. No need to pass it along with the command-line.

Reply:
Yeah - I have just got to that commit. I think it is fine. 
Would you mind to document it a bit in README.md?

Reply:
Sure thing.